packages/postcss/mixin.core.js

```diff
@@ -13,17 +9,46 @@ const getAssetPath = (...args) => trimSlashes(join(...args));
 
 const cssLoaderLocalOptions = {
   modules: {
+    auto: () => true,
     localIdentName: '[folder]-[name]-[local]-[hash:8]',
+    exportLocalsConvention: 'camelCase',
   },
-  localsConvention: 'camelCase',
+  esModule: false,
   sourceMap: process.env.NODE_ENV !== 'production',
 };
 
 const cssLoaderGlobalOptions = {
   modules: false,
+  esModule: false,
   sourceMap: process.env.NODE_ENV !== 'production',
 };
 
+const getPostCssLoader = (browsers) => ({
```

Comment:
Extracting this loader config, because for the `node`-build to succeed, we need all the imports to be resolved properly. Therefore it is applied there, too.

Question's though whether we actually need `postcss-preset-env` for the `node`-build?!?

Reply:
I suppose not. In the Node.js build we need only to compute the exported names, so the import plugin should be enough.

Reply:
Removed it for the Node-build.